qiskit/qobj/pulse_qobj.py

```diff
@@ -174,6 +175,30 @@ def to_dict(self):
                 x.to_dict() for x in self.discriminators]
         return out_dict
 
+    def __repr__(self):
+        out = "PulseQobjInstruction(name='%s', t0=%s" % (self.name, self.t0)
```

Comment:
not really a big deal, but if you use `self.__class__.__name__` instead of hard coding it then one doesn't have to change it if the class name changes.